build-aux/Dockerfile

```diff
@@ -37,28 +37,17 @@ FROM ${envoy} as envoy
 
 FROM ${builderbase} as artifacts
 
-ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/buildroot/bin
```

Comment:
can you provide more context on the why? Its obvious we were doing some magic with PATHs.

Reply:
I've now added a separate commit (d3e9f67c5ee2ba42354a2fbc538fe2547037f92a) just dealing with `PATH` in the Dockerfile.